hmi_sdk/HMI_SDK/Connect/SocketsToSDL.cpp

```diff
@@ -294,28 +283,23 @@ void SocketsToSDL::RunThread()
 		FD_SET(m_Read_Sign, &fdRead);
 
 #ifdef WIN32
-        if (select(0, &fdRead, NULL, NULL, NULL) == SOCKET_ERROR)
+        if (select(0, &fdRead, NULL, NULL, NULL) == SOCKET_ERROR){
 #else
-        if (select(fd_max+1, &fdRead, &fdWrite, NULL, NULL) == SOCKET_ERROR)
+        if (select(fd_max+1, &fdRead, &fdWrite, NULL, NULL) == SOCKET_ERROR){
 #endif
-		{
             goto SOCKET_WRONG;
 		}
```

Comment:
if you use open curly brace with the same line of if(), it's good, but you should keep the close curly brace align with if.
For example:
```
if (condition) {
  foo;
} else {
  bar;
}
```